tests/time_tests/src/timetests/timetest_infer.cpp

```diff
@@ -27,51 +27,54 @@ int runPipeline(const std::string &model, const std::string &device, const bool
         size_t batchSize = 0;
 
         // first_inference_latency = time_to_inference + first_inference
+        SCOPED_TIMER(first_inference_latency);
```

Comment:
Should be moved inside the next visibility scope

Reply:
done , moved inside { } 

however, when executed I see:

[setupvars.bat] OpenVINO environment initialized
[ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -c -f C:\OpenVINO\compiler_cid_latency.conf -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpgm5ojze2
[ INFO ] ERROR: unknown command line flag 'f'
[ INFO ] ========== Completed. Exit code: 1
[ ERROR ] Run of executable 'timetest_infer.exe' failed with return code '1'. Error: ERROR: unknown command line flag 'f'

Reply:
my bad:  -f is not a recognized flag

C:\Users\Local_Admin>cmd /V /C "C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208\setupvars.bat & cd C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\\timetest\scripts & python run_timetest.py timetest_infer.exe -d VPUX -niter 1 -c -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml"
Python 3.9.12
[setupvars.bat] OpenVINO environment initialized
cmd_common: ['C:\\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\\tools\\timetest\\timetest\\scripts\\timetest_infer.exe', '-m', 'C:\\models\\mobilenet-v2\\onnx\\FP16\\mobilenet-v2.xml', '-d', 'VPUX', '-c', '', ''][ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -c   -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpjzl3v24j
[ INFO ] ========== Completed. Exit code: 0
retcode: 0
msg:
raw_data: [[{'full_run': [2554470.0, {'first_inference_latency': [2362160.0, {'time_to_inference': [2296500.0, {'load_plugin': [684983]}, {'create_exenetwork': [1572610.0, {'load_network_cache': [1572610.0]}]}]}, {'first_inference': [65660, {'fill_inputs': [4168]}]}]}]}], {'measurement_unit': 'microsecs'}]
[ DEBUG ] Statistics after run of executable #0: {'full_run': 2554470.0, 'first_inference_latency': 2362160.0, 'time_to_inference': 2296500.0, 'load_plugin': 684983, 'create_exenetwork': 1572610.0, 'load_network_cache': 1572610.0, 'first_inference': 65660, 'fill_inputs': 4168}
[ DEBUG ] Aggregated statistics after full run: {'full_run': {'avg': 2554470.0, 'stdev': 0}, 'first_inference_latency': {'avg': 2362160.0, 'stdev': 0}, 'time_to_inference': {'avg': 2296500.0, 'stdev': 0}, 'load_plugin': {'avg': 684983, 'stdev': 0}, 'create_exenetwork': {'avg': 1572610.0, 'stdev': 0}, 'load_network_cache': {'avg': 1572610.0, 'stdev': 0}, 'first_inference': {'avg': 65660, 'stdev': 0}, 'fill_inputs': {'avg': 4168, 'stdev': 0}}
[ INFO ] Aggregated statistics:
{'create_exenetwork': {'avg': 1572610.0, 'stdev': 0},
 'fill_inputs': {'avg': 4168, 'stdev': 0},
 'first_inference': {'avg': 65660, 'stdev': 0},
 '**first_inference_latency**': {'avg': 2362160.0, 'stdev': 0},
 'full_run': {'avg': 2554470.0, 'stdev': 0},
 'load_network_cache': {'avg': 1572610.0, 'stdev': 0},
 'load_plugin': {'avg': 684983, 'stdev': 0},
 'time_to_inference': {'avg': 2296500.0, 'stdev': 0}}